app/javascript/components/carousel/CarouselFixed.vue

```diff
@@ -1,95 +1,167 @@
+    <p
+      v-for="(slide, index) in slides"
+      :key="slide._uid"
+      @click="scroll(index + 1)"
+      :class="['carousel__nav-item', { 'active': navItemActive(index + 1) }]"
+    >
+      <span class="carousel__nav-item-text">Chapter {{ index + 1 }}</span>
+      <span :class="['carousel__nav-item-icon', { 'active': navItemActive(index + 1) }]" />
+    </p>
+  </div>
 
-    <div class="carousel__nav flex flex-column flex-v-end">
-      <div v-for="(slide, index) in slides" :key="`slide-${index}`" class="carousel__nav-item flex" :class="{ 'active': isActive(index) }">
-        <span class="carousel__nav-text">{{ slide.title }}</span>
-        <span @click="changeSlide(index)" class="carousel__nav-button flex flex-center"> {{ index + 1 }}</span>
+  <div 
+    class="carousel__slides"
+    id="chapter-0" 
+  >
+    <section 
+      v-for="(slide, index) in slides"
+      :key="slide._uid"
+      :id="`chapter-${index + 1}`" 
+      class="carousel__slide"
+      ref="animated-slide"
+    >
+      <div class="carousel__content container">
+        <p class="carousel__subtitle">{{ slide.subtitle}}</p>
+        <h2 class="carousel__title">{{ slide.title }}</h2>
+        <p class="carousel__intro">{{ slide.intro}}</p>
+        <a :href="slide.url" :title="'View chapter: #{slide.title}'" class="carousel__button button--cta">View chapter</a>
       </div>
-    </div>
-  </div>  
+    </section>
+  </div>
+</div>
 </template>
 
 <script>
-  export default {
-    name: 'carousel-fixed',
+import { ScrollToPlugin } from "gsap/ScrollToPlugin"
+import { ScrollTrigger } from "gsap/ScrollTrigger"
+import { gsap } from "gsap"
 
-    props: {
-      slides: {
-        type: Array,
-        required: true
-      }
-    },
+gsap.registerPlugin(ScrollToPlugin, ScrollTrigger)
 
-    data () {
-      return {
-        activeIndex: 0,
-        changingSlide: false,
-        endEvent: ''
-      }
-    },
+export default {
+  name: 'carousel-fixed',
 
-    computed: {
-      title () {
-        return this.slides[this.activeIndex]['title']
-      },
+  props: {
+    slides: {
+      type: Array,
+      required: true
+    }
+  },
 
-      subtitle () {
-        return this.slides[this.activeIndex]['subtitle']
-      },
+  data () {
+    return {
+      activeIndex: 0,
+      atCarouselEnd: false,
+      isActive: false,
+    }
+  },
 
-      intro () {
-        return this.slides[this.activeIndex]['intro']
-      },
+  mounted () {
+    this.scrollTriggerHandlers()
+  },
 
-      url () {
-        return this.slides[this.activeIndex]['url']
-      }
+  methods: {
+    navItemActive (index) {
+      return this.activeIndex == index
     },
 
-    mounted () {
-      this.endEvent = this.getEndEvent()
+    scroll (index) {
+      //For some reason the .to doesn't work 
+      //when scrolling back up so need to -1 off index
+      this.activeIndex = index > this.activeIndex ? index : index - 1
```

Comment:
Did a bit of testing, when selecting a previous chapter the highlighted index still seems to jump to the previous one before it before settling on the chosen chapter. I think it's because `activeIndex` is also being modified separately by the scrolling up motion so it may be conflicting with this line of code.